client/my-sites/customer-home/main.jsx

```diff
@@ -152,6 +160,8 @@ Home.propTypes = {
 	hasWooCommerceInstalled: PropTypes.bool.isRequired,
 	isStaticHomePage: PropTypes.bool.isRequired,
 	isRequestingSitePlugins: PropTypes.bool.isRequired,
+	ssoModuleActive: PropTypes.bool.isRequired,
+	fetchingJetpackModules: PropTypes.bool.isRequired,
 	site: PropTypes.object.isRequired,
 	siteId: PropTypes.number.isRequired,
 	trackViewSiteAction: PropTypes.func.isRequired,
```

Comment:
Minor (non-blocking) nit: can we add these new types in alphabetical order?
```suggestion
	fetchingJetpackModules: PropTypes.bool.isRequired,
	hasWooCommerceInstalled: PropTypes.bool.isRequired,
	isStaticHomePage: PropTypes.bool.isRequired,
	isRequestingSitePlugins: PropTypes.bool.isRequired,
	site: PropTypes.object.isRequired,
	siteId: PropTypes.number.isRequired,
	ssoModuleActive: PropTypes.bool.isRequired,
	trackViewSiteAction: PropTypes.func.isRequired,
```